$OpenBSD$

Setting file permission on config.xml and bookmark.xml to allow user writing.

--- src/RadioTray.py.orig	Fri Mar  9 01:22:31 2012
+++ src/RadioTray.py	Fri Nov 21 10:15:17 2014
@@ -157,11 +157,13 @@ class RadioTray(object):
             else:
                 self.logger.info('Copying default bookmarks file to user directory')
                 copy2(DEFAULT_RADIO_LIST, self.filename)
+                os.chmod(self.filename, 0644)
 
         if not os.access(self.cfg_filename, os.R_OK|os.W_OK):
 
             self.logger.warn('Configuration file not found. Copying default configuration file to user directory')
             copy2(DEFAULT_CONFIG_FILE, self.cfg_filename)
+            os.chmod(self.cfg_filename, 0644)
 
 
     def configLogging(self):
